sample/shared/build.gradle.kts

```diff
@@ -59,6 +62,11 @@ kotlin {
                     implementation(AUTH)
                 }
 
+                with(Dependencies.Moko) {
+                    api(RESOURCES)
+                    // api(RESOURCES_COMPOSE) FIXME: Cannot build the shared module with this dependency
```

Comment:
I am not sure what is the issue with this dependency but I can build the project with this line @AVI5HEK. Can you please provide the logs of the error?

https://github.com/nimblehq/kmm-templates/assets/32578035/64d05879-ab3d-49c2-9772-cbe5ca8942d1